payments-core/src/main/java/com/stripe/android/view/AuthActivityStarterHost.kt

```diff
@@ -43,6 +49,8 @@ sealed class AuthActivityStarterHost {
         override val statusBarColor: Int?
     ) : AuthActivityStarterHost() {
 
+        override val lifecycleOwner: LifecycleOwner = fragment.requireActivity()
```

Comment:
Any concerns with this as the lifecycle owner?

Reply:
Can we guarantee that the activity is a lifecycle owner?

Reply:
Yes, because every activity is. I was just wondering if there are any downsides to using the activity lifecycle instead of the fragment lifecycle. Since we call `fragment.startActivityForResult()` below, I’ll just change the `lifecycleOwner` to also point to the fragment.